.env

```diff
@@ -0,0 +1,2 @@
+NODE_ENV=development
```

Comment:
You should never commit .env files to you repo! Add .env to .gitignore and add .env.example which will be clone of .env but it will have only keys without values

Reply:
added clone files of env and added env files to .gitignore